gapis/api/vulkan/api/vk11structs.api

```diff
@@ -38,61 +38,77 @@
 
 // Structs added in Vulkan 1.1 spec
 
+@structextends("VkBindBufferMemoryInfo")
 class VkBindBufferMemoryDeviceGroupInfo {
+  @values(VK_STRUCTURE_TYPE_BIND_BUFFER_MEMORY_DEVICE_GROUP_INFO)
   VkStructureType sType
   const void*     pNext
   u32             deviceIndexCount
+  @len("deviceIndexCount")
   const u32*      pDeviceIndices
 }
 
 class VkBindBufferMemoryInfo {
+  @values(VK_STRUCTURE_TYPE_BIND_BUFFER_MEMORY_INFO)
   VkStructureType sType
   const void*     pNext
   VkBuffer        buffer
   VkDeviceMemory  memory
   VkDeviceSize    memoryOffset
 }
 
+@structextends("VkBindImageMemoryInfo")
 class VkBindImageMemoryDeviceGroupInfo {
+  @values(VK_STRUCTURE_TYPE_BIND_IMAGE_MEMORY_DEVICE_GROUP_INFO)
   VkStructureType sType
   const void*     pNext
   u32             deviceIndexCount
+  @len("deviceIndexCount")
   const u32*      pDeviceIndices
   u32             splitInstanceBindRegionCount
+  @len("splitInstanceBindRegionCount")
   const VkRect2D* pSplitInstanceBindRegions
 }
 
 class VkBindImageMemoryInfo {
+  @values(VK_STRUCTURE_TYPE_BIND_IMAGE_MEMORY_INFO)
   VkStructureType sType
   const void*     pNext
   VkImage         image
   VkDeviceMemory  memory
   VkDeviceSize    memoryOffset
 }
 
+@structextends("VkBindImageMemoryInfo")
 class VkBindImagePlaneMemoryInfo {
+  @values(VK_STRUCTURE_TYPE_BIND_IMAGE_PLANE_MEMORY_INFO)
   VkStructureType       sType
   const void*           pNext
   VkImageAspectFlagBits planeAspect
 }
 
 class VkBufferMemoryRequirementsInfo2 {
+  @values(VK_STRUCTURE_TYPE_BUFFER_MEMORY_REQUIREMENTS_INFO_2)
   VkStructureType sType
   const void*     pNext
   VkBuffer        buffer
 }
 
+@returnedonly
```

Comment:
This is not returned only.
sType must be a particular value, and IS read by the driver.

Reply:
Changed to returnedchain